### ISSUE TITLE
Añadir "owner" como parametro de búsqueda del GET /api/playlists

### ISSUE THREAD

**Issue by uricantasala** [arnaugarcia/sallefy#173]
Buenas noches,

Quisiera pedir a ver si es posible de que se pudiera añadir el parámetro de búsqueda con GET "owner" para que se puedan listar las playlists de un usuario en concreto.

Yo creo que sería de mucha utilidad y aportaría muchísimo a la API ya que podríamos listar las playlist de un usuario, y eso nos permitiría hacer múltiple pantallas listando información de estos que quedaría perfecto.

Muchísimas gracias,

Saludos,

Para cualquier cosa me tenéis aquí con mi usuario: uricantasala

O bien el correo electrónico: oriol.cantarell@students.salle.url.edu

![Screenshot_2020-04-14-23-43-13-224_com android chrome](https://user-images.githubusercontent.com/6270806/79277651-850d4180-7eaa-11ea-99c5-238f9470c0d1.jpg)

**Comment by arnaugarcia** [comment]
Hola @uricantasala para devolver las playlists de un usuario de puede hacer de la siguiente forma: 
`http://sallefy.eu-west-3.elasticbeanstalk.com/api/users/{login}/playlists`
Saludos!

**Comment by uricantasala** [comment]
Hola @arnaugarcia muchas gracias por indicarme cómo puedo hacer una call para listar las playlists de un usuario en concreto pues es de mucha utilidad.

Quisiera pedir otro planteamiento extra, de acuerdo a lo que dije cuando abrí la issue, lo que se quería plantear, si es posible, era también añadirlo como parámetro de búsqueda del endpoint /API/playlists pues nos serviría para poder acceder de un modo + rápido y con una query intelligente a las playlist recentes (nuevas) o las populares de un usuario o varias combinaciones entre los varios filtros ya que nos ayuda a poder hacer una query intelligente y mostrar unas playlist u otras de un usuario según unos filtros u otros.

Ejemplos:

Lista de playlist populares de un usuario
Lista de playlist nuevas de un usuario
Lista de playlist seguidas de un usuario
Lista de playlist no seguidas de un usuario
...

Muchísimas gracias,

Saludos

😀

**Comment by arnaugarcia** [comment]
Yup! That makes sense! Lo intentaré hacer en el proximo Sprint :D 

Gracias!

**Comment by uricantasala** [comment]
Perfecto @arnaugarcia!

Muchas gracias!!

😀
